openwisp_controller/config/base/vpn.py

```diff
@@ -316,7 +316,7 @@ def get_vpn_server_context(self):
             context[context_keys['server_ip_address']] = self.ip.ip_address
             context[
                 context_keys['server_ip_network']
-            ] = f'{self.ip.ip_address}/{self.subnet.subnet.max_prefixlen}'
+            ] = f'{self.ip.ip_address}/{self.subnet.subnet.prefixlen}'
```

Comment:
Rather than changing the curent behavior, I would add a new variable called `vpn_subnet_<UUID>`, the value should be simply, `str(subnet.subnet)`, which will print something like `10.0.0.0/16`.

Let's leave the current behavior unchanged and add this as a new thing.